backend/src/lib/controllers/ingestapi-ctrl.ts

```diff
@@ -23,6 +23,43 @@ async function createDataset(req: Request, res: Response) {
     return res.status(400).send("Missing required field `data`");
   }
 
+  if (metadata.schema === "Metrics") {
+    for (let i = 0; i < data.length; i++) {
+      //symbol should be valid input or empty
+      if (!["", "Currency", "Percentage"].includes(data[i].percentage)) {
```

Comment:
@AakashBalaji don't we have to localize "Currency" and "Percentage"

Reply:
Thanks for letting me know Triet, just added localizations for currency and percentage